styles.css

```diff
@@ -0,0 +1,164 @@
+@media only screen and (max-width: 600px) {
+  #navbar div {
+    display: flex;
+    gap: 0.5em;
+    padding: 0.5vw 1vw;
+  }
+
+  nav .flex-container {
+    display: flex;
+    flex-wrap: nowrap;
+  }
+
+  .flex-container a {
+    margin: 0;
+    text-align: left;
+  }
+
+  #navbar div a {
+    text-align: center;
+  }
+
+  #navbar header {
+    margin: auto;
+    padding: 0;
+  }
+}
+
+@media only screen and (min-width: 600.1px) {
+  #navbar {
+    float: left;
+    margin: 1em;
+    padding: 1em;
+    width: 15vw;
+    font-family: 'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif;
+    position: fixed;
+    background-color: #a52a2a;
+  }
+
+  #main-doc {
+    float: right;
+    width: 70vw;
+    border-left: 3px solid #000;
+    padding-left: 2vw;
+    padding-right: 2vw;
+    background-color: #f5f5dc;
+  }
+
+  #navbar a {
+    display: block;
+    text-align: left;
+    width: 100%;
+    padding: 1vw 0;
+    font-size: 1.25rem;
+    text-decoration: none;
+    filter: opacity(90%);
+    color: burlywood;
```

Comment:
Please use the hex value for colors to increase accessibility.

Reply:
[OPTIONAL]I suggest you illustrate what `hex` color is to avoid ambiguity

you can use Kindly change from his color to use its `hex` value.
`color:#deb887` to improve accessibility

Reply:
Please kindly change from burlywood color to use its hex value: color: #deb887 to improve accessibility.